saleor/graphql/discount/mutations.py

```diff
@@ -126,9 +116,6 @@ def clean_input(cls, info, instance, input, errors):
         if code == '':
             input['code'] = generate_voucher_code()
         cleaned_input = super().clean_input(info, instance, input, errors)
-        voucher_errors = validate_voucher(cleaned_input)
```

Comment:
Failed python tests are related to this validation of voucher type being removed. Is this intentional?